gnomock_test.go

```diff
@@ -191,6 +191,35 @@ func TestGnomock_withCommand(t *testing.T) {
 	require.NoError(t, r.Close())
 }
 
+func TestGnomock_withEntrypoint(t *testing.T) {
+	r, w := io.Pipe()
+
+	container, err := gnomock.StartCustom(
+		testutil.TestImage,
+		gnomock.DefaultTCP(testutil.GoodPort80),
+		gnomock.WithLogWriter(w),
+		gnomock.WithEntrypoint("/app"),
```

Comment:
I think that this test is not enough, it doesn't prove that `WithEntrypoint` actually works. To prove it, I think we should have 2 subtests: one that does exactly what you suggest, and the other one uses a wrong entrypoint, such as `echo`, and expects to get an error message like `echo: command not found`. What do you think?

Reply:
Sure @orlangure , I have just changed it to be as you described.